playground/mocks/data/idp/idx/login-with-username-password.json

```diff
@@ -0,0 +1,113 @@
+{
+  "version": "1.0.0",
+  "stateHandle": "eyJ6aXAiOiJERUYiLCJhbGlhcyI6ImVuY3J5cHRpb25rZXkiLCJ2ZXIiOiIxIiwib2lkIjoiMDBvenc0cXV1a1ZyUTk5MjkwZzMiLCJlbmMiOiJBMjU2R0NNIiwiYWxnIjoiZGlyIn0..gjeP9IkoC-b_4nkE.OJxNhHkmK5MliPWMcHdQvULa0o5V_Rg1WalQwVwpDW7omgo5Nz6YWY_Eq5FxVV1fy06jtRAn22Er-o4ziYTRslhhsphN_hpqSjoRTqLyDHb2tp2_HCbCPrOIoy1GfQrurDUCDRV2deLy0lEovXc03vjt1DRSljpBrbWHJu9Q-SqwsXhHOLnRBDloNG8oZaLzE1gNsqM-AwwgtF23JwISf5t2NdDRFaOayymisBPaATrGl_FWyiOTPgZFN6Yj0qmLLU_OW0BXO_FJ1-JusCgUGNqXtqWESWqWyt8ibu7xiJ56RtTySysc9u9IOZv5fL8LY20BDnQKbP4x-yep3LW_y9KeQN0L90zHCZ6rkrBMKWwXzujM_roAOUA257D3c42VCajTikp2F_IEWMKr9i91pCyH9AxvWbUZ62ldGH8gWDiS5ONBwYVPeb_JenQBoKscVnbPrqr7cIeBtzMOslcuEFvqdClXk-NY461aUvBDcOynINFiKJdMNYPgKZnNbirjkw_KmyfJNC8e8ed1fQVAOkw5S3TF7J8ZCU0Y_ZJ4T_wuUcsmrkEfV5bRXXO6RMsPDRgvIwbwZ0X3KFWv7rCBGfUbl_NOTKOEX8u4k9dViuDdYpoxwMXaVSvCkOsbgaQgKF_O4NnCQiiSqC-qTRsbZCW24fYBmk8Tub2LsNGWN24Eus_N619SwiEWpbN08pkVffyAZcOakds_p87_d7hRu-XeAM7OHEyWpjGJamL3LwZHbMklcnXStme9h8hSr_Q-Pj5wLToXeMRDri0v5YRszt9w9n6UWkHIxR3tzHZ9uOugVp6KUy8KkDwzgWjuYQvuxlTbruPgbrwAPiv5FaZxB0dk_4QNJZKhEQxzz9Mb-qYvyPJVAQl-sxqyp4BNLmUkwLpZ49JeMP8Wh1ib5CLVNuJtbHqwAQXF1j--CG3ufL62kaOI_v3wc9BiUwuKa10R7aViHShuoOoQEirEdoVbIZV9bNjE4Z6FF5ys3ncEh9DMnqzncx9VoCXCgMhgubSJQ1IOguy56t4WsQxJQE4Rkexq3Zbx9iwBjUE0RZvKrE77IMmFYM1UApovUGnQUT2Cn6Ev1OWr4Cnih0BOvSX_NKXE9R1aKEm-gnHUePDkpW_8Zqg7amITfykhrZkeIPzn_NVwVgXl7JvO-jEt9xRgEdPE8XlJ_a4l5iAli0aYC7WPv4Dg-TYLoFz8lWBcqnf6jO7FdWCjaoSOC5BggVIj3T80Cg4tgy4GYQzq6Fi2hNszppnQpFkSRG6JmW_bWQAAGo8ibKK3U0nyApwfmAKqFtboxnmEtROY7YiDzxjvwMEwhtS2T1Wtf0QpTYlC2C-qOdEN48ZuJxAjxrrJMojm549zbR6eaeRX-HiSfsE2-noeZxuqED15OK9BrPlx7EZ2XgePdD1p_XKf0khJfvGh6Q5Mxc8tbhsFnN6UCFRNNg3ZMDasYLv3YyC48C-294v4zlpyYkY6YJFJ37Tb2KXVhiu5UvFfuJKcVfo0SYZM9O_CZzTTj1tJEXkiJ-BBQfuEnIzoNghNnkPk2iQILgHlYdHQ1R-zLMzEmSJYLDv5JZH2O2ghkJ7SVrroip00ovsygwJkapx_oQU9P6ALciLaC3_sO63ZHL37fg4d86dLxOGVnPpZlOJqbohAPglYsg0gCNSYVrBtZUInYKNJJNYxVUzQCV0ZqNlHUwYv5SulGO2EkuLGki5Uo3X8Ptbb3680Key6njJRL7NO8Su9wS_D3rw8R4fD3lAK9N8zorVTwYlQszSNmaKOttkD1u2Tw1hna1vJC1DUCFOj66UMbG_-xE1LhKUlaTxoLF1YYVMAdiV7CKrzFAh4reaUL_gqrDDp_besFS0oP0aYSPpLAUQnMUGctXfZkFlSdW0gs5kfUFv1plI5Pc6IUK5yIGPolqdDF4c90oYniuJK0f5lcGMe5oDbHVdW8n4jY8gUZ0AwzaDIO4wCZsMQmchdufUx4INtih660Rxzg0runddgnM3W18QplKJjf63D-B6Q_AsdjkphUa4J_uDLVVrJqmEsN7IpQVk-Nsl2cFjV80SmJDMAwIgLu4tEWAehH2qNyoFsnrhJucew2fzsZtADd7ga1r0nLapMFqSKl5wUaW7iUkLAy7HZV4bE6jInA4foAnlW-m28sLlGZTvi8EQEFyqQW2XqtmSeKd-328xEXQqNve-L-WjItHgCBg1kdTXRcE_ZtBAVcURKBvXkyMZSFnUTlzEFbuowQBUkoydTOLVfkXcEaS-ovUryIG1w3NdgjCw7k0S2GA8nYkDQUD2x7wKRTfcomf75rY8T-Zo5l54r6tQg2xv8Tm3NYyShaZLiPhiB6Ez8aGg0HuvZzuoN2u-p3A7uUKTN7tS8dJpeq-MdKoPCp9tToPuae_fcBf7PpQw15GS5wKJCerhc6BlZgkR9NayaV5bCI9HQhBbKr3BZzhIkVY5jXAh73d_Bj_bx.e_A6AQEUcYi7hB6V6_IxFQ",
```

Comment:
you can keep smaller string for stateHandle..

Reply:
guess it can be any string